### PR TITLE
add priority class for extension pods

### PR DESCRIPTION
**Pull request by danielfoehrKn**
**How to categorize this PR?**

/area high-availability
/kind enhancement
/priority normal
/platform azure

**What this PR does / why we need it**:
Adds priority class for extension pods to prevent preemption.
Has same priority as the gardenlet class.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
We do not know which version the Seed is running on

**Release note**:

```improvement operator
Adds priority class for extension pods to prevent preemption.
```

### CODE REVIEW

**Comment by rfranzke** [comment]
Please run `make generate`

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by timebertt — APPROVED** [review]

File: charts/gardener-extension-provider-azure/templates/priorityclass.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: {{ include "priorityclassversion" . }}
+kind: PriorityClass
+metadata:
+  name: gardener-extension-provider-azure
+value: 1000000000
```
Comment (timebertt) [inline charts/gardener-extension-provider-azure/templates/priorityclass.yaml:5]: Hmm, this is the same value, that the gardenlet has. Should we give the extensions less priority than the gardenlet?

Comment (rfranzke) [reply charts/gardener-extension-provider-azure/templates/priorityclass.yaml:5]: I think it makes sense to use the same value. The extension are similarly important like the gardenlet.

**Review by dkistner — APPROVED** [review]

File: charts/gardener-extension-provider-azure/templates/priorityclass.yaml (hunk 1 above)
Comment (timebertt) [reply charts/gardener-extension-provider-azure/templates/priorityclass.yaml:5]: Ok, good :)

**Pull request merged**